### ISSUE TITLE
Ferals should starve

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#58581]
### Is your feature request related to a problem? Please describe.

I am discussing this here before implementing because I know this may or may not be a controversial change.

Feral humans are still alive and not zombified yet. Dehydration is also an issue, but this is something for another discussion perhaps, this one is mainly about starving. They do not eat and will die because of this, unless they are also suddenly cannibals. This will also shift the balance a bit, so no more handgun-wielding ferals late in the game and more demise of humanity.

### Solution you would like.

Add an `upgrades: "mon_feral_starved 21d"` to all ferals or add starved variants to ferals. I prefer the former one because it is less work and when ferals starve, they become more feral and lose their intelligence, meaning they can not use their tools anymore. On the other hand, they will not retain their original drops.

The starved feral is a weaker version of the feral. Since it is physically weaker due to starvation, it has lower stats and is easier to kill, accelerating death greatly so it can "finally" become a zombie.

The starved feral will upgrade into either a normal zombie or a starved one after some time. I prefer the latter. The starved zombie can upgrade into a normal one after a short amount of time.

### Describe alternatives you have considered.

I did not consider dehydration here because it would be boring if all ferals were gone after 3 days.

**Comment by GuardianDll** [comment]
Why do you think they starve? They are left with only basic instincts, but "find the food" and "find the water" is really basic instincts indeed. Also they are, even if is are human, are partially zombie, so we can say they have better digestive system than common human - not perpetum zombile, but still good enough to consume rotted food or even dead bodies of another zeds

**Comment by Gotdamnmiracle** [comment]
I disagree. If a creature is psychotically hunting down and killing its own kind, or even lashing out at others, generally seeking food and water is not a given. The hierarchy of needs, while an interesting idea, generally falls apart when normal functioning of the hypothalamus (the part of the body regulating temperature, thirst, hunger, blood pressure) is not a given.

In my opinion, the "blob psychosis" sounds a lot like mass scale encephalitis. The hordes of dead are probably most of the folks who didn't survive it, and the ones who are still surviving are delirious and violent. Reading case studies for encephalitis you can find a litany of examples for individuals needing to be restrained or made medically comatose due to how violent they were.

A normal functioning appetite is absolutely not a given.

**Comment by RAldrich** [comment]
I disagree. There is no need to provide an explanation for why ferals don't need to eat or drink any more than for why zombies don't need to. The blob powers them.

More importantly, this is a bad change from a game design perspective. Ferals are a distinct and interesting enemy because of their ability to open doors and their access to equipment. Removing them using the evolution timer does nothing but homogenize the enemies and make gameplay less interesting as time goes on.

**Comment by NetSysFire** [comment]
>The blob powers them.

No. The blob powers zombies. Ferals still have "normal" bodily functions. Ferals are not zombies yet but the blob psychosis has affected them gravely. The blob can only radically restructure beings after their death. Ferals are still alive.

**Comment by RAldrich** [comment]
1. I cannot find a design document that sets out the rules for blob behavior that you are claiming here, can you point me to it?

2. I re-assert, this change would be bad for gameplay and make things more boring.

**Comment by NetSysFire** [comment]
1. https://cataclysmdda.org/design-doc/
2. I get that but this means we need alternatives.

**Comment by Gotdamnmiracle** [comment]
> * I cannot find a design document that sets out the rules for blob behavior that you are claiming here, can you point me to it?
> * I re-assert, this change would be bad for gameplay and make things more boring.

I see it as part of a split change. Evolution of zombies exists ingame- why not evolution of ferals. I could see some ferals starving, others turning into zombies, some snapping out of it, and finally others turning into something completely different.

**Comment by RAldrich** [comment]
1. I've read all the design docs linked from there, and do not recall anything about the details of blob mechanics. Do you have a link more specific than the landing page.

To be constructive: mutations are explicitly blob facilitated, so the blob is obviously capable of causing change in living entities.

I think that having ferals evolve into progressively stronger mutant ferals makes a lot of sense, would keep (and add more!) interesting gameplay, and if you really don't want them blob powered, give them all a mutation that lets them eat zombie flesh.

**Comment by NetSysFire** [comment]
>some snapping out of it

I debated bringing this up in here because it is not 100% relevant but I thought about a passive human monster which is meant to be killed. To quote the design doc:

>ranging from mild unsafe practices all the way to people attacking hordes of zombies with a stick

In this case, it is the latter. Perhaps a group of people still holed up in some house, feral enough to not be able to interact with you but normal enough to see you as an "ally" and not attack you. This might prove interesting because this will cause a distraction and make the world a bit more dynamic, even if just a little bit. They will turn into a feral or zombie so you will not encounter those 3 years in.

**Comment by RAldrich** [comment]
Oh hey, I'm just blind - I found the "Effects of Blob Infection" section!

Interestingly enough mutation is called out explicitly:

```
These ferals are not seen as hostile to zombies, and mutate naturally with time just as zombies do.
```

Also relevant:

```
Cognition after the Cataclysm
Over time, most affected people acclimate and return to normal. Some remain somewhat less risk-averse than before (eg players). Those that tended towards high violence also drift back towards normal cognition slowly, although their actions and the memory of them likely leave them changed permanently.
```

For what it's worth, I still don't see anything about Feral physiology. I think that leaves you quite a bit of wiggle room to work with. (And besides, we're still at a pretty low level of NPC/monster resource simulation. I mean hell, all the NPCs just stand in one spot for all of eternity).

**Comment by Gotdamnmiracle** [comment]
I mean, you could always have a few spawn with food on their person. Pretty sure Zombies have more food spawned than ferals, but it would imply them scavenging.

A twist in the other direction with more varied and scavenged goods on their person would be cool rather than the three different flavors (pipe, crowbar, axe). Excluding the specialized variants, of course.

Either starving and evolution or more interesting drops implying scavenging sound like great additions for realism, IMO.

**Comment by GuardianDll** [comment]
> more varied and scavenged goods on their person
can you make a separate feature request? i think someone can hook it up

**Comment by liminalRoar** [comment]
I think it is most logical to assume they still handle the need for food and water. Maybe able to starve, but outside of player intervention, rarely happens.

Or the strongest ferals learn how to survive and adapt and the weak die off. So early game ferals are common and generally "low level" but as the game progresses they become rarer but far more ruthless and dangerous.

Difficult finding the lore on ferals. Is it possible early game many are not 100% aggressive and more prone to hide and survive, but more "evolved" late game ferals being some of the most perceptive, aggressive, and well equipped enemies?

**Comment by AtomicFox556** [comment]
One possible explanation for ferals not completely dying off over time would be that at least some would mutate so as to be able to consume zombie flesh with no issues. This one is pretty much always available.

**Comment by fairyarmadillo** [comment]
If the player can find food, so can ferals, they're eating whatever is left in the wake of the zombie destruction. They're smart enough to open doors and operate flashlights, tasers, and firearms, they can feed themselves. Some of them can even speak a little!

I think this could be revisited when we can have areas getting ambiently looted over time, especially with closed environments like labs. Ferals in those circumstances would either starve or move on, which might result in some interesting behavior where they concentrate in certain settings and even migrate around looking for more food. This same logic could be applied to mutants, with increasing food scarcity driving changes in behavior and maybe even anatomy.

**Comment by NetSysFire** [comment]
>I've always wondered why we don't have any feral nest locations to show how they're surviving

YES! This is a major plot hole in my opinion. Too many things are explained with "because blob".

**Comment by RAldrich** [comment]
I like the direction this proposal is evolving in quite a lot! Adding food and diverse item drops, feral nest locations, and more mutant or mutant like feral evolutions are all strong additions.

With those in place, you would also have more latitude to add starving or otherwise "failed" variations without detracting from existing gameplay.

**Comment by GuardianDll** [comment]
> Too many things are explained with "because blob".

Another explaining is "no one made it yet".
Sometimes i think cdda is a time consuming black hole, the most grind game ever, when you know you will never reach the end
Kinda sad

**Comment by Gotdamnmiracle** [comment]
> I like the direction this proposal is evolving in quite a lot! Adding food and diverse item drops, feral nest locations, and more mutant or mutant like feral evolutions are all strong additions.
> 
> With those in place, you would also have more latitude to add starving or otherwise "failed" variations without detracting from existing gameplay.

It occurs to me with the addition of more diverse mutants this MAY be the end result of a feral mutating down a path. All we would need to do is bridge that gap.

**Comment by NetSysFire** [comment]
>Another explaining is "no one made it yet". There is still too much things to do

Yes, but the problem is that helping out is difficult since changes like this need to be "approved" by the lore people (they make sure that the lore they have in mind and in the design doc matches the newly added content) which are currently kevin and erk I think. Both are pretty busy and no one gets paid to do this anyways, I do not blame them for not reading through the literal thousands of suggestions yet I wish there was a way to fix these lore issues.

So yes I would love to help out by fixing the stuff that annoys me.

**Comment by fairyarmadillo** [comment]
> > I like the direction this proposal is evolving in quite a lot! Adding food and diverse item drops, feral nest locations, and more mutant or mutant like feral evolutions are all strong additions.
> > With those in place, you would also have more latitude to add starving or otherwise "failed" variations without detracting from existing gameplay.
> 
> It occurs to me with the addition of more diverse mutants (the shelled mutant, bestial mutant, etc.) this MAY be the end result of a feral mutating down a path. All we would need to do is bridge that gap.
> 
> Gimme a couple hours. I'm gonna throw all these up as three separate feature requests. Wish I was a capable coder and could knock this out on my own.

Those mutants are not ferals, they're mutants who have gone crazy from having their brains too mutated. They are not part of the zombie/feral faction, and will fight them if they see each other.

I-am-Erk has mentioned that feral mutants should be a rare and super-powerful threat.

**Comment by Gotdamnmiracle** [comment]
The description of ferals literally mentions people attacking waves of zombies with a stick. I think you're deriving too much from code. Because they are buddies with the zombies right now doesn't mean that won't change in a week. Literally "snapping out of it" is how the player comes to be.

Also portal storms cause mutations that are codedly indistinct from human mutagen. Literally everyone except a handful of humans were exposed to the portal storm that started all of this.

Mutation seems like more of a continuum- regardless of how it occurs.

**Comment by fairyarmadillo** [comment]
I don't think a request is needed unless someone wants to add something that conflicts with lore. The primary thrust of this conversation doesn't involve mutations so I don't see a reason.

**Comment by Gotdamnmiracle** [comment]
> I don't think a request is needed unless someone wants to add something that conflicts with lore. The primary thrust of the OP's post doesn't involve mutations so I don't see a reason unless you've got a specific and separate idea.

I think feral evolution is reasonable to discuss. It's just out of the scope of "ferals starving", don't you think?

**Comment by RAldrich** [comment]
I agree, if the "mutant ferals" bit is controversial, let's split that out and address it separately. New map locations for feral "nests" and updates to drop lists can be done in parallel and no one seems opposed to them so far.

**Comment by fairyarmadillo** [comment]
> As a point of note, feral mutation is a thing: [Discord link](https://discord.com/channels/598523535169945603/598535827169083403/972935259584868382)
> 
> > **kevingranade:** [response to ongoing conversation] mutation stuff should not be common at all. The materials are pretty readily available just because of the way our spawning logic works in legacy code and content but you should almost never encounter another mutant especially one that keeps diving deeper and deeper into mutations
> > **I-am-erk:** On that note though I would like to still have people who mutate randomly just as animals do
> > **I-am-erk:** Yusuke and Makaila in the refugee center for example
> > **I-am-erk:** It can still be quite rare, especially for non-ferals. An interesting point might be exploring why ferals often mutate and non ferals rarely do.
> > **kevingranade:** Yes sure but more as a risk than an intentional transhumanism thing
> > **I-am-erk:** Also lends itself to some in universe anti-mutant sentiment since for every makaila there's ten frenzied cannibal monsters
> 
> [Discord link too](https://discord.com/channels/598523535169945603/610001025755119630/857007966464901191)
> 
> > **I-am-erk:** Most of the time [the exodii] just shoot mutants on sight, once we have systems for that.
> > **I-am-erk:** A highly mutated player will be unable to talk to exodii regardless of faction standing.
> > **I-am-erk:** That's not even prejudice or anything, they have good reason for that as we'll see once feral evolutions start being a thing
> > **actual-nh:** There's too much blob in you?
> > **I-am-erk:** Nah, evolved ferals will be similar to threshold mutants, and dangerous on a scale with or exceeding advanced zombies.

There's some discussion here if you want to keep at it. It's clear that we're supposed to get feral mutants at some point, but they're a distinct thing from the hostile lab mutants and mutated but sane humans. A feral mutant would hang out with zombies and do zombie things, the lab mutants don't. There's even a whole little mini-plot about it.

There's been no discussion at the top that I've seen of whether ferals can become sane again, and becoming temporarily super-aggressive during the Cataclysm doesn't necessarily qualify you for ferality. To be a feral, you have to go so far off the deep end that the zombies think you're one of them.

**Comment by Gotdamnmiracle** [comment]
Last comment on the subject here, I promise:

Right! I was trying to suggest that feral mutants exist nearer to the center of the mutation continuum which in my mind looks something like this...

1. pure human
2. humans exposed to blob and portal event (most people)
3. further exposure (hanging out in more portal storms, unintentional radiation exposure)
4. Mostly mutated (most of pre-threshold mutations/post threshold)
5. Full mutant (insane, violent, inhuman)

I imagine the survivors and ferals existing between 2 and 4 on that continuum, unless deliberately pushing one direction or the other. 

Zombies are different as the blob seems to be deliberately and gracefully changing their traits, rather than randomly doing so. This is how it's able to maintain control- its not just taking a claw hammer to the genetic code and messing with brain chemistry, so to speak. Seems to me there are ways ferals could mutate outside blob intervention or mutagen, namely portal exposure.

**Comment by RAldrich** [comment]
Sub topic: what distinguishes a feral from a particularly brutal bandit? I assume it's the social origination and how the zombie faction responds to them.

Answering this is mostly relevant to creating new "nest" locations. LMOEs already have multiple ferals, which implies some degree of coordination. Is there a limit on group size? Should there be?

**Comment by NetSysFire** [comment]
>Is there a limit on group size? Should there be?

Re this and the nest size, there appears to be no answer in the design documentation. Only Erk and perhaps Kevin can answer this.

>what distinguishes a feral from a particularly brutal bandit?

I assume it is because their mind is clouded. A brutal bandit does so by choice in our case. In the real ~world~Cataclysm, according to the design documentation, everyone becomes a bit more aggressive and ignores risks.

> Some may be strongly enough affected to be almost indistinguishable from ferals, but not aligned with zombies.
The remaining quarter see a heavy increase in aggressive, violent behaviour, often completely out of character. The worst of these, about 1/20 of the population, become “ferals”, a form of living zombie.

(from https://cataclysmdda.org/lore.html)

But a bloodthirsty bandit is still far more intelligent than a feral.

**Comment by RAldrich** [comment]
Intelligence as a limiting factor is a nice scope limiter. Feral map specials are probably disorganized, no use of appliances, barricades, or traps (unless they were set up before going feral), no advanced weapons (even the ferals with guns don't seem to know how to reload), no vehicles, no organized stockpiles or evidence of long range planning.

By contrast any and all of these things could go into a corresponding bandit update.

**Comment by Gotdamnmiracle** [comment]
That makes sense, but again, what is a cunning feral? If they're the same as a feral, but with a different loadout then they should be renamed. 

I assumed a cunning feral was a delusional, violent, and speechless person, but had a mostly intact recognition of objects and planning. In regards to intelligence wolves and dogs can organize ambushes, chimpanzees and octopi can use tools (up to and including loading a gun if they understand it). The trap setting is important because I'd like to answer the question of what they're doing in the time between when they were turned and when the player sees them. Scavenging would only be part of the answer, I'd think. For what it's worth, a wildly paranoid person might also set traps.

Admittedly, I'm using pseudo-humanoids (like orcs or kobolds) as kind of a guide here.  But for you to lose your ability to plan and use tools you'd have to lose far more of your faculties (they should appear far more delirious or confused) which isn't currently represented in game.

I argue that cunning feral leading attacks on the player, while requiring a lot of intelligence compared to zombies, is still reasonable and adds a lot of cool flavor.

**Comment by fairyarmadillo** [comment]
"Living zombie" doesn't sound anywhere near as intelligent as a chimpanzee or octopus.

**Comment by NetSysFire** [comment]
I think we need the assistance of @kevingranade and/or @I-am-Erk here since they are deeply involved with the lore and could answer some uncertainities and questions which came up in the discussion here.

As much I'd love crude tribes of ferals which get more advanced and vicious later on, I want a yes or no before going further with this. Another question I have is about the need to sleep for ferals, so can they just roam around endlessly, scavenging for food or do they settle down in nests?

**Comment by Theawesomeboophis** [comment]
> >I've always wondered why we don't have any feral nest locations to show how they're surviving
> 
> YES! This is a major plot hole in my opinion. Too many things are explained with "because blob".

They could be stealing food from houses like most players do, they seem to be nomadic, travelling with the zombies without the zombies bothering them, kind of like the Whisperers from Walking Dead without the zombie skin suits.

**Comment by RAldrich** [comment]
> >They could be stealing food from houses like most players do
> 
> Then the food spawns need to be adjusted to account for (feral) looting. I *think* there is already a "remove food on adjacent tiles" function but that would only partially work because ferals might not be in houses too often.

I think it would be much more straightforward to simply change their loot tables. I don't think it's worth the effort to have them actually pick up food from the map. Plus that's kind of a low reward change, because it's hard to make visible to the player: you probably wouldn't even notice it's happening unless you can see it, and if you can see it you have an angry feral in your face.

**Comment by I-am-Erk** [comment]
If anything, all ferals are particularly good at is finding food, including other survivors. The lore already includes some hints about this, they eat anything and everything. It wouldn't be unreasonable for them to be *able* to starve, if cut off from resources, but it's not at all a given. Rather, I would love to see feral "nests" as map extras you could find, littered with food scraps and weird vestiges of their civilized life.

**Comment by I-am-Erk** [comment]
In addition to my comment above I realize I should add: I'm not sure it's worth the code to have ferals be able to starve, even if it can happen from a lore perspective. It's not something that should happen frequently, and it's likely to involve a fair bit of overhead to manage.

**Comment by chaoticidealism** [comment]
I like the idea of having starving and dehydrated ferals as variants. That makes sense.

But I don't think they should die of starvation or dehydration. The madness they're under does seem to allow them a primitive sort of cognition. They can use tools, for example--throw rocks at you. And the zombies don't bother them. 

Ferals should feel hunger and thirst, and should respond to those instincts in a basic way. So they should eat food when they see it. Maybe it's a pack of crackers; maybe it's a dead squirrel. See food, eat it. Same with water. Maybe it's Gatorade, maybe it's vodka. Looks drinkable; chug it. Maybe it's bleach, and they join the zombies.

The way I figure it, ferals still have some of their thinking power, but not that much of it. I'd put them at the level of a monkey--not a chimp, just a standard little long-tailed, clever-fingered monkey. And being able to eat and drink makes sense, even if most of what they feel is fury.

That said, there should be other things that can happen to ferals, and would create variants. Maybe they get badly injured, and they're dragging themselves around and biting with their filthy teeth. Maybe they're weakened by bad food and tainted water, and they're sick--maybe sick with things you can catch if you get close enough, including, say, influenza and other communicable diseases. Imagine strep throat spreading like wildfire among the ferals... and you better get some antibiotics into you if you catch it, because it can turn to scarlet fever and kill you dead. (Shouldn't be as severe as a wound infection, but still.)

And perhaps, here and there, a feral human gets the knack for living in this rage-filled, primitive state. Maybe they started out strong and tough and nimble. They got lucky and nothing killed them early on. They didn't die of disease or infection and they found a good source of food. Ferals like that could become a force to be reckoned with, because unlike the dead, they can still learn. Maybe they become good brawlers, when they survive. Maybe they learn to make noise and call a horde of zombies. They'd be rare, but one in every few hundred might make it in that fashion.

**Comment by AideeAmei** [comment]
The way I perceive feral humans, if something is so far out there that zombies don't attack it, it is probably so changed, that the processes known to us as means of standard living no longer apply.

Perhaps at least their digestive habits could shift towards pure carnivore. So you could actually find ferals scouring the woods for some prey.

But if they eat, they should be able to do more human stuff, like for example short boosts of running towards their prey. Because it seems funny that the influence of the Blob would let them open doors but it would keep them from running.

Also as I see it, the Blob doesn't "power" anything, it merely controls 3D vehicles for multidimensional beings humans are when the ties to the human soul (higher dimensions) is either weak (poor-minded) or severed (dead, comatose etc.).

The Blob is like a big pack of undead souls that doesn't get a body of its own and needs to steal bodies to live experience. Its goal is not to live a life and find happiness, but to bring chaos by severing control from as many souls as possible. Like universe is powered by consensus of living beings, Blob is a consensus of dead beings that strives to un-power the universe, hence the deadly character of its "creations". The way I see the Cataclysm, this process is active on other planets, too.

So the way I see it, it shouldn't really power anything, it just hijacks the normal processes and steals the meat from the bacteria's feast table to infuse it with otherworldly control. The processes going on in those beings are not natural, but entirely possible. Just not on normal untipped quantum scale.

Relevant question to this debate is whether the processes going on in "hosts" are enough to feed them. And I'm perfectly fine with either "it is" or "it's not". Just... if you're considering the latter, it'd make more sense to me to provide the means to sustain these beings in a way that is logically believable.

**Comment by kevingranade** [comment]
Ferals are eating off camera. I'm not sure why there's all this argument about blob powers and such when it is absolutely trivially easy to survive if you simply don't have to fight zombies and stay in towns,  which is exactly what ferals do.
